recipes/libelfin/all/CMakeLists.txt

```diff
@@ -0,0 +1,67 @@
+cmake_minimum_required(VERSION 3.1)
+
+include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
+project(${CONAN_PACKAGE_NAME} VERSION ${CONAN_PACKAGE_VERSION})
+conan_basic_setup()
+
+file(GLOB_RECURSE elf_sources source_subfolder/elf/*.cc)
+set(elf_headers
+    source_subfolder/elf/common.hh
+    source_subfolder/elf/data.hh
+    source_subfolder/elf/elf++.hh)
+file(GLOB_RECURSE dwarf_sources source_subfolder/dwarf/*.cc)
+set(dwarf_headers
+    source_subfolder/dwarf/data.hh
+    source_subfolder/dwarf/dwarf++.hh
+    source_subfolder/dwarf/small_vector.hh)
+
+add_custom_command(
+    OUTPUT source_subfolder/elf/to_string.cc
+	COMMAND ${CMAKE_COMMAND} -E echo '// Automatically generated' > to_string.cc
+	COMMAND ${CMAKE_COMMAND} -E echo '// DO NOT EDIT' >> to_string.cc
+    COMMAND ${CMAKE_COMMAND} -E echo >> to_string.cc
+	COMMAND ${CMAKE_COMMAND} -E echo '\#include \"data.hh\"' >> to_string.cc
+    COMMAND ${CMAKE_COMMAND} -E echo '\#include \"to_hex.hh\"' >> to_string.cc
+    COMMAND ${CMAKE_COMMAND} -E echo >> to_string.cc
+	COMMAND ${CMAKE_COMMAND} -E echo 'ELFPP_BEGIN_NAMESPACE' >> to_string.cc
+    COMMAND ${CMAKE_COMMAND} -E echo >> to_string.cc
+    COMMAND python3 enum-print.py -u --hex --no-type --mask shf --mask pf -x loos -x hios -x loproc -x hiproc < data.hh >> to_string.cc
+	COMMAND ${CMAKE_COMMAND} -E echo 'ELFPP_END_NAMESPACE' >> to_string.cc
+    DEPENDS source_subfolder/elf/enum-print.py source_subfolder/elf/data.hh
+    WORKING_DIRECTORY source_subfolder/elf)
+
+add_custom_command(
+    OUTPUT source_subfolder/dwarf/to_string.cc
+	COMMAND ${CMAKE_COMMAND} -E echo '// Automatically generated' > to_string.cc
+	COMMAND ${CMAKE_COMMAND} -E echo '// DO NOT EDIT' >> to_string.cc
+	COMMAND ${CMAKE_COMMAND} -E echo >> to_string.cc
+	COMMAND ${CMAKE_COMMAND} -E echo '\#include \"internal.hh\"' >> to_string.cc
+	COMMAND ${CMAKE_COMMAND} -E echo >> to_string.cc
+	COMMAND ${CMAKE_COMMAND} -E echo 'DWARFPP_BEGIN_NAMESPACE' >> to_string.cc
+	COMMAND ${CMAKE_COMMAND} -E echo >> to_string.cc
+    COMMAND python3 ../elf/enum-print.py < dwarf++.hh >> to_string.cc
+    COMMAND python3 ../elf/enum-print.py -s _ -u --hex -x hi_user -x lo_user < data.hh >> to_string.cc
```

Comment:
Don't use `python3`, but [`FindPythonInterp`](https://cmake.org/cmake/help/latest/module/FindPythonInterp.html).
Does the script work on python2? If not, set the correct options and fail if necessary.

Reply:
Yes, Python 3 is required. Fixed.